src/components/menu/MenuList.tsx

```diff
@@ -1,31 +1,35 @@
 import React, { ReactChild, ReactNode } from 'react'
 
-import ListSubheader from 'cozy-ui/transpiled/react/MuiCozyTheme/ListSubheader'
 import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
 import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import ListSubheader from 'cozy-ui/transpiled/react/MuiCozyTheme/ListSubheader'
 
-type MenuListProps = {
+interface MenuListProps {
   title: string
   children: ReactChild | ReactChild[] | ReactNode | ReactNode[]
 }
 
 export const MenuList = ({ title, children }: MenuListProps): JSX.Element => (
   <li>
-    <List className="u-pb-1">
-      <ListSubheader className="u-mb-1">{title}</ListSubheader>
+    <List>
+      <ListSubheader>{title}</ListSubheader>
 
-      {Array.isArray(children)
-        ? children
-            .filter(value => value)
-            .map((child, index, array) => (
-              <React.Fragment key={index}>
-                {child}
-                {index !== array.length - 1 && (
-                  <Divider component="li" variant="inset" />
-                )}
-              </React.Fragment>
-            ))
-        : children}
+      <li>
```

Comment:
`<li />` are used because `<ListItem />` has too many specific styles that are cumbersome to remove.
The main reason to add these tags is to keep a semantic and valid HTML5.